modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase

```diff
@@ -0,0 +1,238 @@
+			}
+		}
+
+		task ("Then two separate line items for each SKU/UOM combination are searched") {
+			CommerceInventory.openInventoryAdmin();
+
+			CommerceNavigator.searchEntry(entryName = "MIN93015");
+
+			for (var index : list "1,2") {
+				AssertTextEquals(
+					key_cellName1 = "MIN93015",
+					key_cellName2 = "uomKey${index}",
+					key_columnNumber = 2,
+					locator1 = "CommerceEntry#TABLE_CELL_BY_COLUMN_ORDER_NO_TITLE",
+					value1 = "uomKey${index}");
+			}
+
+			Pagination.viewResults(results = "Showing 1 to 2 of 2 entries.");
+		}
+
+		task ("And the default quantities for UOM1 are On Hand:140, Available:140, On Order:0, Incoming:0, for UOM2 are all 0") {
+			for (var uomQuantityList : list "3:140,4:140,5:0,6:0") {
+				var uomIndex = StringUtil.extractFirst(${uomQuantityList}, ":");
+				var uomQuantity = StringUtil.extractLast(${uomQuantityList}, ":");
+
+				AssertTextEquals(
+					key_cellName1 = "MIN93015",
+					key_cellName2 = "uomKey1",
+					key_columnNumber = ${uomIndex},
+					locator1 = "CommerceEntry#TABLE_CELL_BY_COLUMN_ORDER_NO_TITLE",
+					value1 = ${uomQuantity});
+			}
+
+			for (var uomQuantityList : list "3:0,4:0,5:0,6:0") {
+				var uomIndex = StringUtil.extractFirst(${uomQuantityList}, ":");
+				var uomQuantity = StringUtil.extractLast(${uomQuantityList}, ":");
+
+				AssertTextEquals(
+					key_cellName1 = "MIN93015",
+					key_cellName2 = "uomKey2",
+					key_columnNumber = ${uomIndex},
+					locator1 = "CommerceEntry#TABLE_CELL_BY_COLUMN_ORDER_NO_TITLE",
+					value1 = ${uomQuantity});
+			}
+		}
+
+		task ("And the admin can edit the warehouse inventory for each SKU/UOM combination") {
+			for (var uomQuantityList : list "1:10,2:20") {
+				var uomIndex = StringUtil.extractFirst(${uomQuantityList}, ":");
+				var uomQuantity = StringUtil.extractLast(${uomQuantityList}, ":");
+
+				CommerceInventory.editInventoryByWarehouse(
+					index = ${uomIndex},
+					quantityOnHand = ${uomQuantity},
+					sku = "MIN93015",
+					warehouse = "Italy,United States - Northeast,United States - Southwest");
+
+				for (var index : list "2,4") {
+					CommerceInventory.viewInventoryByWarehouse(
+						index = ${index},
+						inventory = ${uomQuantity},
+						warehouse = "Italy,United States - Northeast,United States - Southwest");
+				}
+
+				task ("And the SKU and UOM Key are shown in the header") {
+					AssertElementPresent(
+						key_title = "MIN93015 uomKey${uomIndex}",
+						locator1 = "Header#H3_TITLE");
+				}
+			}
+		}
+	}
+
+	@description = "This is a test for COMMERCE-12324. The admin can verify the inventory information from the SKU/UOM combination on the product SKU page"
+	@priority = 5
+	test VerifyUOMInventoryInfoOnSKUPage {
+		property portal.acceptance = "true";
+
+		task ("Given a Minium Site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("When the admin goes into MIN93015's sku panel -> Inventory tab on the product admin page") {
```

Comment:
When the admin goes to the MIN93015's SKU panel -> Inventory tab on the product admin page